tests/data/basic_exotic_section_types.swc

```diff
@@ -0,0 +1,6 @@
+# Comment line
+1 5 0 0 0 6 -1
+3 8 3 4 5 6 1
```

Comment:
If the first row is ID it cannot be duplicated, so it should be 1,2,3... Unless this file is used to catch an error.

Reply:
Doesn't this mean that the SWC reader is doing something strange?

Reply:
It is still using the custom reader ?

Reply:
Sorry forgot this issue. What do you mean? I have an swc reader for TMD so if that's what you mean yes. This file is still wrong though (you cannot have two lines with the same ID in an swc file).